tokio/tests/unwindsafe.rs

```diff
@@ -2,6 +2,22 @@
 #![cfg(feature = "full")]
 
 use std::panic::{RefUnwindSafe, UnwindSafe};
+use tokio::task::spawn_blocking;
+
+#[tokio::test]
+async fn futures_are_unwind_safe() {
+    unwind_safe_future(|| async {
+        let _ = spawn_blocking(|| {}).await;
+    })
+    .await
+}
+
+async fn unwind_safe_future<F, Fut>(_: F)
+where
+    F: FnOnce() -> Fut,
+    Fut: std::future::Future<Output = ()> + std::panic::UnwindSafe,
+{
+}
```

Comment:
That's a pretty convoluted way to test this.

Reply:
I agree, I altered it to test the actual change 🙂